Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/SecurityAndComplianceV2.yml

```diff
@@ -1,7 +1,4 @@
 category: Email
-sectionorder:
-- Connect
-- Collect
```

Comment:
I believe this is still needed. @BEAdi do you know if this is still relevant?

Reply:
It is. Lets add it back